support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.js

```diff
@@ -147,6 +147,7 @@ export type RegularPaymentRequest = {|
   telephoneNumber: Option<string>,
   promoCode?: Option<string>,
   deliveryInstructions?: Option<string>,
+  freeTrialLength?: Option<number>,
```

Comment:
For the record, `freeTrialLength` has the same net effect now within Digital Subscriptions as `firstDeliveryDate` does within paper subscriptions - they both effectively delay the customer acceptance date in the subscription order.